### PR TITLE
Minor bug fixation in R

### PR DESCRIPTION
**Pull request by gschang**
I fixed a minor runtime bug in R source code.

### CODE REVIEW

**Comment by chrisamiller** [comment]
+1

File: lib/perl/Genome/Model/Tools/Analysis/Coverage/CoveragePlot.R [hunk 1]
```diff
@@ -1123,6 +1123,9 @@ max1 <- 0;
 for (i in 1:length(b$stats))
     max1 <- max(max1, as.numeric(unlist(strsplit(b$stats[i], ","))));
 
+# if with extreme distribution
+if (max1 < 5) max1 <- 5;
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Analysis/Coverage/CoveragePlot.R]: Would it make sense to replace `max1 <- 0` before the `for` loop with `max1 <- 5`, and then not add this conditional?

File: lib/perl/Genome/Model/Tools/Analysis/Coverage/CoveragePlot.R [hunk 2]
```diff
@@ -1088,7 +1088,7 @@ xlab3 = "Average coverage depth of each ROI";
 ylab3 = "Density";
 
 xlab4 = NULL;
-ylab4 = "Coverage (%)";
+ylab4 = "Coverage (%)";###
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Analysis/Coverage/CoveragePlot.R]: Does the `###` have a significance?

**Comment by tmooney** [comment]
+1.  I've made a few comments, but this looks fine to me!

**Comment by gschang** [comment]
Hi, Tom and Chris.

I am going to merge pull request. I got Tom's comments and I will reflect new changes in the next upgrade. Since this module was just developed, I will keep updating and fixing it with feedback again.

Thanks a lot.

**Pull request merged**